en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,70 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
+
+## API Format
```

Comment:
Since we are going with title-style capitalization for the page title and sentence-style capitalization for all other subsection headings, might have to follow the same for all subsection headings on this page.

Reply:
done.